### DISCUSSION TITLE
Custom Java Classes

### DISCUSSION THREAD

**Discussion by avigosain** [nextgenhealthcare/connect#5595] [General]
How does mirth manages the objects of custom classes that are called from mirth. e.g if i have a java class to perform simple  steps into  Redis db lets says connect, read and disconnect. So does these three steps are performed for every message that passes thru the channel that call this java class? what is the best practice here to avoid creating connection and closing connection? anyone tried connection pooling etc?

**Comment by wi4seo** [comment]
Hey Guys,

FREE Registration for Mirth Unconference 2023 is Now Opened!

To know more, please visit the event page and register yourself: https://wi4.org/mirth-connect-unconference-feb-2023/

Stay Tuned for More Updates!​

​Thanks
